src/main/java/tds/common/NoContentResponse.java

```diff
@@ -0,0 +1,26 @@
+package tds.common;
+
+import java.util.Optional;
+
+/**
+ * Contains errors (if any) when responding without data (e.g. when the response would be a 204 - No Content).
+ */
+public class NoContentResponse {
```

Comment:
It isn't clear to me why this class is needed (comment around it in the pause exam PR), but if we do keep it I think it should be in the `web` package since its only use is for the Rest controllers.